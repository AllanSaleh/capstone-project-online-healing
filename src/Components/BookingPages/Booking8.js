import React from 'react';

const Booking8 = () => (
  <div className="h-firstsection px-sides pt-navbar">
    <h1 className="text-title uppercase mb-4">
      your request has been submitted!
    </h1>
    <h3 className="text-subtitle opacity-50 text-justify">
      You will receive an email guiding you to book a date and time soon.
    </h3>
    <div className="flex flex-col justify-evenly items-center h-2/3 max-w-3xl my-16 mx-auto p-8 shadow-md">
      <h3 className="text-3xl capitalize">request submitted</h3>
      <p className="text-subtitle text-center">
        you will receive a confirmation email soon
        <br />
        please keep an eye on your mail.
      </p>
      <button
        type="button"
        className="uppercase bg-blue-dark text-subtitle px-4 py-2 min-w-min w-1/3 rounded-md border hover:border-blue-dark hover:bg-white hover:text-blue-dark transition-all duration-300"
      >
        back to home
      </button>
    </div>
  </div>
);

export default Booking8;
