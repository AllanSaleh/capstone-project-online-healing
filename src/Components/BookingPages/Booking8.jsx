import React from 'react';
import { Link } from 'react-router-dom';

export default function Booking8() {
  window.scrollTo(0, 0);
  return (
    <div className="lg:h-firstsection px-sides pt-navbar">
      <h1 className="text-2xl lg:text-title uppercase my-8">your request has been submitted!</h1>
      <h3 className="text-md lg:text-subtitle opacity-50 text-justify">
        You will receive an email guiding you to book a date and time soon.
      </h3>
      <div className="flex flex-col justify-evenly items-center h-1/2 lg:h-2/3 max-w-3xl my-16 mx-auto p-8 shadow-md">
        <h3 className="text-xl lg:text-3xl capitalize">request submitted</h3>
        <p className="text-md lg:text-subtitle text-center">
          you will receive a confirmation email soon
          <br />
          please keep an eye on your mail.
        </p>
        <Link to="/">
          <button
            type="button"
            className="uppercase bg-blue-dark text-md lg:text-subtitle px-4 py-2 lg:w-64 rounded-md border hover:border-blue-dark hover:bg-white hover:text-blue-dark transition-all duration-300"
          >
            back to home
          </button>
        </Link>
      </div>
    </div>
  );
}
