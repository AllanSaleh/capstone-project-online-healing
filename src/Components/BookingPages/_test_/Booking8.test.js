import * as React from 'react';
import Booking8 from "../Booking8";
import * as renderer from "react-test-renderer";

it('Booking8 renders correctly', () => {
  const tree = renderer
    .create(<Booking8/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});