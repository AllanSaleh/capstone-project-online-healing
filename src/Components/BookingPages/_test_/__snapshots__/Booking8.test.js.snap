// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="h-firstsection px-sides pt-navbar"
>
  <h1
    className="text-2xl lg:text-title uppercase mb-4"
  >
    your request has been submitted!
  </h1>
  <h3
    className="text-md lg:text-subtitle opacity-50 text-justify"
  >
    You will receive an email guiding you to book a date and time soon.
  </h3>
  <div
    className="flex flex-col justify-evenly items-center h-1/2 lg:h-2/3 max-w-3xl my-16 mx-auto p-8 shadow-md"
  >
    <h3
      className="text-xl lg:text-3xl capitalize"
    >
      request submitted
    </h3>
    <p
      className="text-md lg:text-subtitle text-center"
    >
      you will receive a confirmation email soon
      <br />
      please keep an eye on your mail.
    </p>
    <button
      className="uppercase bg-blue-dark text-md lg:text-subtitle px-4 py-2 lg:min-w-min lg:w-1/3 rounded-md border hover:border-blue-dark hover:bg-white hover:text-blue-dark transition-all duration-300"
      type="button"
    >
      back to home
    </button>
  </div>
</div>
`;
